### ISSUE TITLE
Bitdefender 2019 blocking upload of presentations to servers using Lets Encrypt certificate

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#6030]
It appears that Bitdefender 2019 is blocking upload of slides to BigBlueButton servers configured using a Lets Encrypt certificate

   https://groups.google.com/d/msg/bigbluebutton-users/HGTXHuJL_ig/xVD5CBSVBAAJ

When the upload fails, the user has the message

> "Add Files to Your Presentation" with "Progress: Upload completed. Please wait while we convert the document." "LOADING 0%"

Need to duplicate and confirm and, if possible, give a better error message that indicates the document was not uploaded (or detect if a zero length document was uploaded).

**Comment by prlanzarin** [comment]
@ffdixon can I close this (seems stale)?

**Comment by PhMemmel** [comment]
I'm not the one pinged here, but I am using current Bitdefender and I've been using it since I started BBB first (Aug '20). I'm running all my BBB servers with LetsEncrypt and have never experienced any issues with uploading presentation to them neither I have received any feedback about problems from my users. I doublechecked if I have this bitdefender check mentioned in the google groups discussion active and I do, so probably Bitdefender realized LE-Certificates are not "unsafe". Probably not an issue (anymore).
